### PR TITLE
Add 1 n pictures

### PR DESCRIPTION
**Pull request by lucienicolier**
I made the change we talked about yesterday. Is that OK for you?

### CODE REVIEW

File: en/project-management/add-1-n-pictures.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+Add 1:n pictures
+====================
+
+You can add one or more pictures to a feature. Below you find an exemple how to proceed.
+
+Creating tow tables
```
Comment (signedav) [inline en/project-management/add-1-n-pictures.rst]:
```suggestion
Creating two tables
```

File: en/project-management/add-1-n-pictures.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+* id
+* apiary_id (UUID)
+* path (TEXT)
+* ...
+
+Relations
+----------
+
+create a relation with:
```
Comment (signedav) [inline en/project-management/add-1-n-pictures.rst]:
```suggestion
Create a relation with:
```

File: en/project-management/add-1-n-pictures.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+       :width: 600px
+       :alt: Relations
+
+Widgets
+--------
+
+apiary
+~~~~~
+
+Set default value of id to ``uuid()``. No need to show it in the form.
```
Comment (signedav) [inline en/project-management/add-1-n-pictures.rst]:
```suggestion
Set the default value of id to ``uuid()``. No need to show it in the form.
```

File: en/project-management/add-1-n-pictures.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+.. container:: clearer text-center
+
+    .. image:: /images/add-1-n-pictures-widgets2.png
+       :width: 600px
+       :alt: widgets2
+
+picture
+~~~~~~~
+
+Set widget type of path to ``attachment``
```
Comment (signedav) [inline en/project-management/add-1-n-pictures.rst]:
```suggestion
Set the widget type of path to ``attachment``
```

**Comment by lucienicolier** [comment]
I did the corrections is it ok now? 
Thanks for the review

**Comment by signedav** [comment]
Thanks @lucienicolier

**Pull request merged**